src/pages/HomeDashBoard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import NoteState from "../context/NoteState";
+import Inputfield from "../comonents/InputField";
+import TablePage from "../comonents/TablePage";
+import './homepage.css'
+
+function HomeDashBoard() {
+  return (
+    <div>
+      <div className="home_div">
+        <NoteState>
+          <h1>ToDo App</h1>
+          <Inputfield />
```

Comment:
input field represent particular input field , 
make here a form and use onSubmit property.